content/pages/features.html

```diff
@@ -71,9 +71,12 @@ <h3>A Test Plan is a collection of Test Cases</h3>
                     </ul>
                 </div>
 
-                <div style="float: right; clear:right;margin-bottom: 5rem;">
+                <div class="col-2"></div>
+
+                <div style="margin-bottom: 5rem;" class="col-3">
```

Comment:
Revert this change because it skews the layout of the image and follow @ivo0126 advice to move these style definitions as CSS classes.